roles/configure-nginx/templates/backend-upstreams.conf

```diff
@@ -1,4 +1,4 @@
-upstream backends {
-    server 127.0.0.1:8888;
+upstream backends-{{ kci_app_name }} {
+    server 127.0.0.1:{{ kci_app_port }};
```

Comment:
There appeared to be an issue when deploying a second instance on kernelci.org, some queries were being handled by the wrong one.  Please check if the config files were deployed correctly.